package.json

```diff
@@ -13,7 +13,8 @@
     "docs:update": "yarn build && node scripts/generate-api-docs.js",
     "lint": "eslint . --ext .ts",
     "prepare": "yarn build",
-    "test": "jest --no-cache"
+    "test": "yarn lint && yarn test:jest",
```

Comment:
I'd _vaguely_ prefer something like what we do in Ember apps:

```
"test": "npm-run-all lint test:*",
```

Reply:
Done.